RULES/xilinx/bitstream.go

```diff
@@ -9,16 +9,16 @@ import (
 )
 
 type BuildFileScriptParams struct {
-	Out        core.Path
-	PartName   string
-	BoardName  string
-	Name       string
-	IncDir     core.Path
-	Timing     core.Path
-	BoardFiles []core.Path
-	Ips        []core.Path
-	Constrs    []core.Path
-	Rtls       []core.Path
+	Out          core.Path
+	PartName     string
+	BoardName    string
+	Name         string
+	IncDir       core.Path
+	BoardFiles   []core.Path
+	Ips          []core.Path
+	Constrs      []core.Path
+	Rtls         []core.Path
+	OutOfContext bool
```

Comment:
This is the parameters for building the synthesis scripts. Bitstream and out-of-context targets use the same template, since 99% of the code is the same. The bitstream target itself has no out-of-context option. When building the synthesis script, the OutOfContext parameter is set to false (line 95)